.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       run: flake8 --show-source --statistics
     - name: Run tests
       run: pytest
-  lint:
+  reformat:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
```

Comment:
what does this action do ?
will it fail if the code is not formatted correctly ?

if it checks for changes after applying black then it's actually a linter ?
and if it just apply changes (e.g. reforamt) maybe we don't need it as ci ?

Reply:
this action will fail on bad formating, so it's actually a linter action,
can we rename it "blake" , because we already do a basic linting using flake8